docs/maintainers/README.md

```diff
@@ -98,15 +98,15 @@ In these cases:
 
 ## Becoming a Repository Maintainer
 
-Repository Maintainers currently consist entirely of Microsoft employees.
+Repository Maintainers currently consist mostly of Microsoft employees.
 It is expected that over time, regular trusted contributors to the PowerShell repository will be made Repository Maintainers.
-Eligibility is heavily dependent on the level of contribution and expertise: individuals who contribute in meaningful ways to the project will be recognized accordingly.
+Eligibility is heavily dependent on the level of contribution and expertise: individuals who contribute in meaningful ways consistently to the project will be recognized accordingly.
 
-At any point in time, a Repository Maintainers can nominate a strong community member to become a Repository Maintainer.
-Nominations should be submitted in the form of [RFCs][RFC-repo] detailing why that individual is qualified and how they will contribute.
-After the RFC has been discussed, a unanimous vote by the PowerShell Committee will be required for the new Repository Maintainer to be confirmed.
+At any point in time, the existing Repository Maintainers can nominate a strong community member to become a Repository Maintainer.
+Nominations are brought to the PowerShell Committee to understand the reasons and justification.
```

Comment:
My opinion is that the maintainers should be unanimous.  My thinking is that the Committee doesn't approve, but has veto power which I think a simple majority may be ok (I don't expect it to happen often/ever).  @joeyaiello @BrucePay thoughts?